elements/lisk-transactions/src/utils/create_base_transaction.ts

```diff
@@ -37,3 +40,16 @@ export const createBaseTransaction = ({
 		senderPublicKey,
 	};
 };
+
+export const getSignaturesBytes = (signatures: ReadonlyArray<string>) => {
+	if (signatures?.length) {
+		// tslint:disable-next-line: no-unnecessary-callback-wrapper
+		const signaturesBuffer = signatures.map(signature =>
+			hexToBuffer(signature),
```

Comment:
Isn't this missing the prepend of booleans as per LIP? https://github.com/LiskHQ/lips/blob/master/proposals/lip-0017.md#signatures-serialization ?